app/javascript/components/molecules/LeaseUpSidebar.js

```diff
@@ -54,10 +55,20 @@ const LeaseUpSidebar = ({ isLoading, statusItems, onSaveClicked, onChangeStatus,
 
   const numberOfStatusesToDisplay = showingAllStatuses ? statusItems.length : MAX_UPDATES_TO_SHOW_DEFAULT
 
+  const header = (title) => (
+    <>
+      <div className='padding-top--3x show-large-up'>
```

Comment:
used padding here instead of margin just because we don't currently margins only go up to 3x. Can add it though if this should be a margin instead of padding

Reply:
not a big deal to me for margin vs padding.